roles/openshift_examples/tasks/main.yml

```diff
@@ -4,6 +4,11 @@
     src: "examples/{{ content_version }}/"
     dest: "{{ examples_base }}/"
 
+- name: Modify registry paths if registry_url is not registry.access.redhat.com
+  shell: >
+    find {{ examples_base }} -type f | xargs -n 1 sed -i 's|registry.access.redhat.com|{{ registry_host }}|g'
```

Comment:
Since registry_host could be user provided, we should probably use the `quote` filter here.